recipe/run_test.sh

```diff
@@ -36,24 +34,14 @@ if [[ $PKG_NAME == "openmpi-mpicxx" ]]; then
 fi
 
 if [[ $PKG_NAME == "openmpi-mpifort" ]]; then
-
   command -v mpifort
   mpifort -show
 
-  command -v mpif90
-  mpif90 -show
-
-  command -v mpif77
-  mpif77 -show
-
-  mpif77 $FFLAGS $LDFLAGS helloworld.f -o helloworld_f
```

Comment:
Why these changes? Should these paths not be tested?

Reply:
In Open MPI,  compiler wrappers (C/C++/Fortran) are symlinks to `opal_wrapper`. As I explained in the MPICH update PR, `mpifort` is the "modern" name for the Fortran compiler wrapper. This is a good move, newer Fortran standards are quite backward compatible with previous one, there is no point on keep adding compiler wrappers named `mpif95`, `mpif03`, `mpif08` and so on, just use `mpifort` from now on. Let's follow best practices, it is really annoying seeing `mpirun` still used after 30 years of `mpiexec` being standardized in MPI-2, so we should stop using `mpif77`/`mpif90` and move on.

Using just `mpifort` for testing both f77 and f90 sources looks enough to me. If you do not agree, I can add additional tests for `mpif77`/`mpif90`, though IMHO these extra tests would provide very little value.